fdbclient/NativeAPI.actor.cpp

```diff
@@ -4759,7 +4767,10 @@ ACTOR Future<Void> getRangeStreamFragment(Reference<TransactionState> trState,
 			req.limitBytes = std::numeric_limits<int>::max();
 			// leaving the flag off for now to prevent data fetches stall under heavy load
 			// it is used to inform the storage that the rangeRead is for Fetch
-			// req.isFetchKeys = (trState->taskID == TaskPriority::FetchKeys);
+			// TODO: update to FETCH once the priority multi lock is used.
+			req.options = trState->readOptions;
+			req.options.type = ReadType::NORMAL;
```

Comment:
Streaming range is intended to be used for other things, it just happens to only be used for 1 thing (data fetch) so far.  We shouldn't override the options type here, just use what the transaction has set.

Reply:
The FetchKeys actor in StorageServer uses a transaction to read data via either get range or streaming get range, so it should set the read type and cache flag.